bugsnag-plugin-android-ndk/src/main/jni/deps/parson/parson.h

```diff
@@ -145,7 +145,7 @@ int json_object_dothas_value_of_type(const JSON_Object *object, const char *name
 JSON_Status json_object_set_value(JSON_Object *object, const char *name, JSON_Value *value);
 JSON_Status json_object_set_string(JSON_Object *object, const char *name, const char *string);
 JSON_Status json_object_set_number(JSON_Object *object, const char *name, double number);
-JSON_Status json_object_set_long(JSON_Object *object, const char *name, long number);
+JSON_Status json_object_set_long(JSON_Object *object, const char *name, int64_t number);
```

Comment:
Might be worth separately renaming all of these function to use `_integer()` instead of long, since that is the underlying type now, and it took a bit to spot that the type no longer matched

Reply:
This is a good suggestion 👍